package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "@apollo/client": "3.5.10",
-    "@ensembl/ensembl-genome-browser": "0.2.0",
+    "@ensembl/ensembl-genome-browser": "0.2.1-test.2",
```

Comment:
*NOTE TO SELF*: Change the version to `0.2.1` when the related PR in `ensembl-genome-browser` is merged.

Reply:
Done ✅